src/README.md

```diff
@@ -0,0 +1,56 @@
+Multi-Entry GiST and Multi-Entry SP-GiST Indexing
+=================================================
+
+This directory contains an implementation of the Multi-Entry GiST and Multi-Entry SP-GiST access methods for PostgreSQL. They are a variation of the GiST and SP-GiST indexes that allow for more efficient indexing of complex and composite data types.
+
+The extension on its own adds
+
+   - a Multi-Entry GiST access method and an implementation of a multi-entry R-tree for the PostgreSQL `multirange` and `path` types.
+   - a Multi-Entry SP-GiST access method and an implementation of a multi-entry Quadtree for the PostgreSQL `multirange` and `path` types.
+
+For more advanced uses of the Multi-Entry GiST and Multi-Entry SP-GiST indexes, see the example use-cases below.
+
+Dependencies
+------------
+- [PostgreSQL 16](https://www.postgresql.org/)
+
+Installation
+------------
+Compiling and installing the extension
+```
+make
+sudo make install
+```
```

Comment:
Not sure how the README files should be organised, but the compilation process should be run in the top-level directory, not in `src/`. Putting these commands here will be confusing.

Reply:
We should probably put the compilation and installation process in the top-level directory README. The `CREATE INDEX` examples can then be here (in `src/`), and we should add README files to `src/mgist/` and `src/mspgist` explaining how to create new operator classes for these access methods (extractValue, logic of consistent methods, etc.).